### PR TITLE
Di 21784 cpi get files in folder

### PR DESCRIPTION
**Pull request by saargt**

### CODE REVIEW

File: cpi-side/pfs.service.ts [hunk 1]
```diff
@@ -84,10 +84,10 @@ export abstract class PfsService
 		const canonizedPath = downloadKeyRes.startsWith('/') ? downloadKeyRes.slice(1) : downloadKeyRes;
 		const whereClause = `Key="${canonizedPath}"`;
 		const res = await this.pfsGetter.getObjects(whereClause);
-		if (res.Objects.length === 1) 
+		if (res.length === 1) 
 		{
 			console.log(`File Downloaded`);
-			return res.Objects[0];
+			return res[0];
```
Comment (ShirMarciano) [inline cpi-side/pfs.service.ts:90]: The return object of this.pfsGetter.getObjects(whereClause) was changed?

Comment (saargt) [reply cpi-side/pfs.service.ts:90]: Yes, this is to allow the refactor later on. Both server side and cpi side's DAL will return AddonData.

**Comment by ShirMarciano** [comment]
@saargt  need to do something before I merge it?

**Comment by saargt** [comment]
Done @ShirMarciano.  go ahead and merge :)

**Pull request merged**